packages/translations/src/i18next/i18next.ts

```diff
@@ -167,6 +167,12 @@ export const localize = (string: string, values?: Record<string, unknown>) => {
     return i18n.t(crc32(string).toString(), { defaultValue: string, ...values });
 };
 
+export const useLocalize = () => {
+    useTranslation();
+
+    return { localize };
```

Comment:
what does it solve just to return the `localize` function from this hook?

Reply:
@shahzaib-deriv Because `useTranslation` hook will trigger to repaint this hook when `i18n.language` changes